packages/suite/src/components/suite/landing/ConnectDevice/index.messages.ts

```diff
@@ -22,6 +22,10 @@ const definedMessages = defineMessages({
         id: 'TR_DEVICE_NOT_RECOGNIZED_TRY_INSTALLING',
         defaultMessage: 'Device not recognized? Try installing the {link}.',
     },
+    TR_BRIDGE: {
+        id: 'TR_BRIDGE',
+        defaultMessage: 'Trezor Bridge',
+    },
```

Comment:
unnecessary: "Trezor Bridge" is a application name and it will not be transalated